content/oci/storage.go

```diff
@@ -106,6 +106,22 @@ func (s *Storage) Push(_ context.Context, expected ocispec.Descriptor, content i
 	return nil
 }
 
+// Delete removes the descriptor blob from the storage
```

Comment:
The documentation is required to be reworded.
```suggestion
// Delete removes the target blob.
```